src/components/CharacterDetails.tsx

```diff
@@ -0,0 +1,29 @@
+import { useQuery } from "@apollo/client";
+import { GET_CHARACTER } from "../graphql/Characters";
+import CharacterDetailsCard from "./CharacterDetailsCard";
+import { CharacterDetailsProps } from "../utils/types";
+import Loading from "./Loading";
+
+const CharacterDetails: React.FC<CharacterDetailsProps> = ({
+  selectedCharacter,
+}) => {
+  const { data, loading, error } = useQuery(GET_CHARACTER, {
+    variables: {
+      id: selectedCharacter,
```

Comment:
you can use the skip prop for omit the first fetching of the data and handle this query as a lazy query one, that must be the approach in order to don't have a preselected character

Reply:
Sounds good to me, I will implement in that way the next time.